tool/update_goldens.sh

```diff
@@ -4,9 +4,14 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-export DART_VM_OPTIONS="-DUPDATE_GOLDENS=true"
+if [ $1 = "--stable" ]; then
```

Comment:
Lets verify you are on stable if you claim you are. E.g. 
```
flutter --version
Flutter 1.5.2-pre.9 • channel unknown • unknown source
```
indicates I am not on stable.